.github/workflows/build.yaml

```diff
@@ -157,6 +158,7 @@ jobs:
         uses: ./.github/actions/parallel-ctest-containers
         with:
           container: ${{fromJSON(needs.d.outputs.p)[matrix.platform].image}}
+          options: --privileged
```

Comment:
This is a noop. `parallel-ctest-containers` is a custom action that runs all these non-parallel tests in parallel by launching a bunch of docker containers. I'm not sure what the status of the current support for running the tests in parallel now is.

**If tests can be run in parallel:**
You should remove usage of `parallel-ctest-containers` and fold together Tests & NP Tests. You will need to do some other upkeep to ensure logs continue to be uploaded properly on failed tests, etc.

**If tests cannot be run in parallel:**
You need to add `--privileged` to the `parallel-ctest-containers` impl where it launches docker containers
https://github.com/AntelopeIO/leap/blob/c36f68d00bc397b55a8d9a2479b5290d513d19f0/.github/actions/parallel-ctest-containers/main.mjs#L31